pkg/api/index.go

```diff
@@ -376,12 +376,6 @@ func (hs *HTTPServer) buildAdminNavLinks(c *models.ReqContext) []*dtos.NavLink {
 		})
 	}
 
-	if hasAccess(ac.ReqGrafanaAdmin, ac.EvalPermission(ac.ActionServerStatsRead)) {
```

Comment:
@kalleep We're merging the licensing and stats tabs, should this permission check be moved to the API for stats perhaps? We do have a front end permissions check for stats info but might be a good idea to have it on back end as well.

Reply:
There should be a check in the stats api already. If we dont add a navlink for stats anymore this is fine